contracts/interfaces/IAirBroFactory.sol

```diff
@@ -4,6 +4,8 @@ pragma solidity ^0.8.16;
 interface IAirBroFactory {
     function treasury() external returns (address);
 
+    function trustedRelayer() external returns (address);
```

Comment:
This is forgotten.
Also AirbroFactory should inherit from IAirBroFactory, and it is not at the moment.

Reply:
@bxmmm1 when I try to import IAirBroFactory.sol in AirbroCampaignFactory, it shows an error due to it now me having to override the `admin()` function described in the interface, but that is actually a public admin variable in the factory contract derived from AirdropAdmin contract that is imported. 

Not sure what would be the best fix. Would it be making a external getAdmin() function on purpose even though admin var is already public? And changing the code in IAirBroFactory to state: 
`    function getAdmin() external returns (address);
`